Dockerfile

```diff
@@ -57,6 +57,9 @@ RUN chmod 777 /var/run/mysqld
 
 COPY ./backend/yangcatalog-rotate /etc/logrotate.d/yangcatalog-rotate
 
+COPY elasticsearchIndexing/pyang_plugin/json_tree.py /backend/lib/python3.9/site-packages/pyang/plugins/.
+COPY elasticsearchIndexing/pyang_plugin/yang_catalog_index_es.py /backend/lib/python3.9/site-packages/pyang/plugins/.
+
```

Comment:
Context has changed, this need to be set to ./backend/elasticsearchIndexing ....